red_october/red_october.py

```diff
@@ -1,7 +1,8 @@
 # -*- coding: utf-8 -*-
-# Copyright 2016 LasLabs Inc.
+# Copyright 2016-2017 LasLabs Inc.
 # License MIT (https://opensource.org/licenses/MIT).
 
+import json
```

Comment:
Shitty namespace of `loads` and `dumps` basically. `import json.loads` doesn't work either 😦 

```
>>> import json.loads
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named loads
```